docs/misc/circuit_breaker.md

```diff
@@ -0,0 +1,41 @@
+### Circuit breaker
+
+Circuit breaker are enabled by default for each endpoint and the settings can be adjusted globally, but also per API definition.
+
+#### Configuration
+Set the following environment variables to overwrite the settings globally:
```

Comment:
IMO we must not allow overriding CB settings globally - this should be proxy-related setting

Reply:
What if I want to configure one CB for all proxies and I have more than 300 proxies? I’ll have to go one by one and configure be same thing on all?